prize-tool/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+authors = ["Solana Maintainers <maintainers@solana.com>"]
+edition = "2018"
+name = "solana-tds-prize-tool"
+description = "Solana Tour de SOL prize tool"
+version = "0.20.0"
```

Comment:
might as well start at 1.0.0 here, there's no tie in with the main repo version really